src/service/asset_meta_service.rs

```diff
@@ -157,11 +157,10 @@ impl<'a> AssetMetaRepository for AssetMetaService<'a> {
             self.append_messages(&[assess_custom_fee(
                 Coin {
                     denom: verifier_detail.onboarding_denom.clone(),
-                    // The payment detail now contains the originally-specified fee to be charged,
-                    // but halved.  Charge a fee to the onboarding requestor using double the value
```

Comment:
so nice to be able to get rid of all this complicated 2x and /2 verbiage everwhere